pkgdb2/api/packages.py

```diff
@@ -1061,13 +1094,14 @@ def api_package_critpath():
     elif str(critpath).lower() in ['0', 'false']:
         critpath = False
 
-    if pkgnames and branches:
+    if pkgnames and branches and namespace:
```

Comment:
Below, you write:

```python
         if not pkgnames:
             detail.append('pkgnames: This field is required.')
         if not branches:
             detail.append('branches: This field is required.')
```

If ``namespace`` is going to be required here, it should be added to that error handling also.